src/main/java/org/opensearch/security/dlic/rest/api/RateLimitersApiAction.java

```diff
@@ -75,18 +75,14 @@ public class AuthFailureListenersApiAction extends AbstractApiAction {
         )
     );
 
-    protected AuthFailureListenersApiAction(
```

Comment:
@cwperks @shikharj05 @DarshitChanpura what do you folks think about renaming this class to RateLimitersApiAction, but keeping the current routes as `/authfailurelisteners*`? This should give us the flexibility of later on adding other routes such as `/apitokenratelimiter*` to support other use cases?